internal/keppel/errors.go

```diff
@@ -128,6 +128,7 @@ type RegistryV2Error struct {
 // AsRegistryV2Error tries to cast `err` into RegistryV2Error. If `err` is not a
 // RegistryV2Error, it gets wrapped in ErrUnknown instead.
 func AsRegistryV2Error(err error) *RegistryV2Error {
+	//nolint:errorlint // the casted variable is used
```

Comment:
This is a case where errors.As could maybe actually be used without making the implementation less clear.